pkg/proxy/winkernel/proxier.go

```diff
@@ -163,13 +166,15 @@ type endpointsInfo struct {
 	hns             HostNetworkService
 }
 
-//Uses mac prefix and IPv4 address to return a mac address
+//Uses mac prefix and IP address to return a mac address
 //This ensures mac addresses are unique for proper load balancing
-//Does not support IPv6 and returns a dummy mac
 func conjureMac(macPrefix string, ip net.IP) string {
 	if ip4 := ip.To4(); ip4 != nil {
 		a, b, c, d := ip4[0], ip4[1], ip4[2], ip4[3]
 		return fmt.Sprintf("%v-%02x-%02x-%02x-%02x", macPrefix, a, b, c, d)
+	} else if ip6 := ip.To16(); ip6 != nil {
+		a, b, c, d := ip6[15], ip6[14], ip6[13], ip6[12]
```

Comment:
this may have collisions https://play.golang.org/p/g1PuMck4UU4
discussed here https://github.com/containernetworking/plugins/pull/414#issuecomment-634459459, maybe just a comment mentioning this possible risk of mac collision should be enough

Reply:
Sure, I will add a comment.